tools/kokoro/unit-test/trampoline.sh

```diff
@@ -1,4 +1,5 @@
-# Copyright 2018 Google Inc. All Rights Reserved.
+#!/bin/bash
+# Copyright 2017 Google Inc.
```

Comment:
probably insignificant, why did the year change from 2018 to 2017?

Reply:
I perhaps should have noted this in the description -- I've not found a good way to maintain file history while copying files. And, I deleted a few files as a part of this, and some of the copied files are compared to deleted files. So, unfortunately, the diff is kind of terrible. I've yet to find a simple way to make the diffs non-terrible when copying kokoro configs (any insight on this would be appreciated).

So, the year changed from 2018 to 2017 because this is a diff of two unrelated files.